[2023-07-26 03:28:19,411] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: fetch_exchange_rates_dag.create_total_exchange_rates_table manual__2023-07-26T03:28:14.157380+00:00 [queued]>
[2023-07-26 03:28:19,418] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: fetch_exchange_rates_dag.create_total_exchange_rates_table manual__2023-07-26T03:28:14.157380+00:00 [queued]>
[2023-07-26 03:28:19,419] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-26 03:28:19,419] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-26 03:28:19,420] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-26 03:28:19,433] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): create_total_exchange_rates_table> on 2023-07-26 03:28:14.157380+00:00
[2023-07-26 03:28:19,437] {standard_task_runner.py:52} INFO - Started process 4455 to run task
[2023-07-26 03:28:19,440] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fetch_exchange_rates_dag', 'create_total_exchange_rates_table', 'manual__2023-07-26T03:28:14.157380+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/my_calc_rates.py', '--cfg-path', '/tmp/tmpqn9n6ldl', '--error-file', '/tmp/tmpd0undulu']
[2023-07-26 03:28:19,442] {standard_task_runner.py:80} INFO - Job 17: Subtask create_total_exchange_rates_table
[2023-07-26 03:28:19,508] {task_command.py:371} INFO - Running <TaskInstance: fetch_exchange_rates_dag.create_total_exchange_rates_table manual__2023-07-26T03:28:14.157380+00:00 [running]> on host 6e11a261c833
[2023-07-26 03:28:19,565] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=marina_z
AIRFLOW_CTX_DAG_ID=fetch_exchange_rates_dag
AIRFLOW_CTX_TASK_ID=create_total_exchange_rates_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-26T03:28:14.157380+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-26T03:28:14.157380+00:00
[2023-07-26 03:28:19,570] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/my_calc_rates.py", line 115, in create_total_exchange_rates_table
    cur.execute(create_table_query)
psycopg2.errors.SyntaxError: syntax error at or near "'exchange_rates_BTC'"
LINE 4:             SELECT * FROM f'exchange_rates_BTC'
                                   ^

[2023-07-26 03:28:19,580] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=fetch_exchange_rates_dag, task_id=create_total_exchange_rates_table, execution_date=20230726T032814, start_date=20230726T032819, end_date=20230726T032819
[2023-07-26 03:28:19,587] {standard_task_runner.py:97} ERROR - Failed to execute job 17 for task create_total_exchange_rates_table (syntax error at or near "'exchange_rates_BTC'"
LINE 4:             SELECT * FROM f'exchange_rates_BTC'
                                   ^
; 4455)
[2023-07-26 03:28:19,613] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-07-26 03:28:19,637] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
